Jenkinsfile.integration

```diff
@@ -50,16 +36,16 @@ node('lisk-integration') {
 			try {
 				deleteDir()
 				checkout scm
-
-				sh '''
-					# Cycle postgres, install deps and prepare config
-					sudo service postgresql restart
-					rsync -axl -e "ssh -oUser=jenkins" master-01:/var/lib/jenkins/lisk/node_modules/ "$WORKSPACE/node_modules/" || true
-					npm install
-					cp test/data/config.json test/data/genesis_block.json .
-				'''
+				// Cycle postgres, restore chache install deps and prepare config
+				sh 'sudo service postgresql restart'
```

Comment:
I remember we avoid running any thing on Jenkins with sudo privileges. @Nazgolze did refactor to remove sudo from all scripts. Please check with him if its really good to use.

Reply:
Restarting postgres is the one sudo command that the lisk user is allowed to do on the Jenkins machines

And since the job locks the resource and this one machine only does the integration job, I don't see any problem doing this here